tree/dataframe/inc/ROOT/RDF/ColumnReaderUtils.hxx

```diff
@@ -75,24 +75,24 @@ struct RColumnReadersInfo {
 /// Create a group of column readers, one per type in the parameter pack.
 template <typename... ColTypes>
 std::array<std::shared_ptr<RDFDetail::RColumnReaderBase>, sizeof...(ColTypes)>
-MakeColumnReaders(unsigned int slot, TTreeReader *r, TypeList<ColTypes...>, RColumnReadersInfo &colInfo,
-                  const std::string &variationName = "nominal")
+GetColumnReaders(unsigned int slot, TTreeReader *r, TypeList<ColTypes...>, const RColumnReadersInfo &colInfo,
```

Comment:
I checked the body of this function. We take some class members (`fColNames`, `fLoopManager`, `fColRegister`) from the `colInfo` object and then pass them to `GetColumnReader`.

Afair `GetColumnReader` modifies these, i.e. the `std::move(treeColReader)`. Hence `RColumnReadersInfo` should not be `const` (hence windows does not complain)?

Reply:
The `treeColReader` in `std::move(treeColReader)` is a local variable created in `GetColumnReader`, so it's unrelated to the constness of `RColumnReadersInfo`.

I don't see where windows complains about violating constness either?